src/ray/common/ray_syncer/ray_syncer.h

```diff
@@ -142,6 +142,11 @@ class RaySyncer {
   /// \param message The message to be broadcasted.
   void BroadcastMessage(std::shared_ptr<const RaySyncMessage> message);
 
+  /// Function to force a broadcasting of messages come from a component_id
+  ///
+  /// \param component_id The component which need to send the broadcasting messages
+  void BroadcastMessage(ray::rpc::syncer::RayComponentId component_id);
```

Comment:
What about BroadcastComponentState? I found `BroadcastMessage ` a bit confusing

Reply:
or it could be `BroadcastState`

Reply:
Actually `BroadcastSnapshot` might be better

Reply:
I think I choose message because maybe it's not a snapshot. For example, GC command is not a snapshot.

Could you share some idea about why `BroadcastMessage` confuse you? We might be able to get a better name.

Reply:
For this interface that accepts the component id, it broadcasts the snapshot from the implementation? (`void BroadcastMessage(ray::rpc::syncer::RayComponentId component_id);`). I think `BroadcastMessage` with `component_id` is confusing to me here because it doesn't require any message (and it is not clear to me the relationship between message & component_id)

Reply:
For example;
```
placement_group_resource_manager_->CommitBundles(bundle_specs);
ray_syncer_.BroadcastMessage(syncer::RayComponentId::RESOURCE_MANAGER);
```

When I read this part, I wasn't sure about what it meant by broadcasting the message of the resource manager. I assume what happens is the resource state is synced to the cluster?

Reply:
Got it. Let me try to update this.